src/test/java/bio/terra/app/utils/ControllerUtilsTest.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.app.utils;
+
+import bio.terra.app.controller.exception.ValidationException;
+import bio.terra.common.category.Unit;
+import bio.terra.model.JobModel.JobStatusEnum;
+import bio.terra.model.JobModel;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.junit.Assert.assertEquals;
+
+@Category(Unit.class)
+public class ControllerUtilsTest {
+
+    @Test(expected = ValidationException.class)
+    public void testValidateEnumerateParamsOffset() {
+        ControllerUtils.validateEnumerateParams(-1, 1);
+    }
+
+    @Test(expected = ValidationException.class)
+    public void testValidateEnumerateParamsLimit() {
+        ControllerUtils.validateEnumerateParams(0, -1);
+    }
+
+    @Test
+    public void testValidateEnumerateParamsOk() {
+        ControllerUtils.validateEnumerateParams(0, 1);
+    }
+
+    @Test
+    public void testJobToResponse() {
+        var jobModel = new JobModel();
+        jobModel.setId("id");
```

Comment:
FWIW, you can define these fluently doing something like:
```
var jobModel = new JobModel()
    .("id");
```

Not saying you have to do it here, just an FYI